src/helpers/debugDeep.ts

```diff
@@ -1,18 +1,27 @@
 import type { ReactTestRendererJSON } from 'react-test-renderer';
-import format from './format';
+import format, { FormatOptions } from './format';
+
+export type DebugOptions = {
+  message?: string;
+} & FormatOptions;
 
 /**
  * Log pretty-printed deep test component instance
  */
 export default function debugDeep(
   instance: ReactTestRendererJSON | ReactTestRendererJSON[],
-  message?: string
+  options?: DebugOptions | string
 ) {
+  const message = typeof options === 'string' ? options : options?.message;
+
+  const formatOptions =
+    typeof options === 'object' ? { mapProps: options.mapProps } : undefined;
```

Comment:
Should prevent crash if user passes `null`, because `typeof null === 'object'`

```suggestion
    typeof options === 'object' ? { mapProps: options?.mapProps } : undefined;
```